### PR TITLE
Enhance traces for File offset issues

### PR DESCRIPTION
**Pull request by sagisin**
This commit adds changes to enhance or give context to the generic error traces related to file offset errors.

### CODE REVIEW

File: oem/ibm/libpldmresponder/file_io_by_type.cpp [hunk 1]
```diff
@@ -88,8 +88,10 @@ int FileHandler::transferFileData(const fs::path& path, bool upstream,
         size_t fileSize = fs::file_size(path);
         if (offset >= fileSize)
         {
-            std::cerr << "Offset exceeds file size, OFFSET=" << offset
-                      << " FILE_SIZE=" << fileSize << "\n";
+            std::cerr
+                << "FileHandler::transferFileData: Offset exceeds file size, OFFSET="
+                << offset << " FILE_SIZE=" << fileSize
+                << "FILE_HANDLE=" << fileHandle << "\n";
```
Comment (ArchanaKakani) [inline oem/ibm/libpldmresponder/file_io_by_type.cpp:94]: Add space before "FILE_HANDLE="

File: oem/ibm/libpldmresponder/file_io_by_type.cpp [hunk 2]
```diff
@@ -206,8 +208,9 @@ int FileHandler::readFile(const std::string& filePath, uint32_t offset,
     size_t fileSize = fs::file_size(filePath);
     if (offset >= fileSize)
     {
-        std::cerr << "Offset exceeds file size, OFFSET=" << offset
-                  << " FILE_SIZE=" << fileSize << "\n";
+        std::cerr << "FileHandler::readFile:Offset exceeds file size, OFFSET="
+                  << offset << " FILE_SIZE=" << fileSize << "FILE_HANDLE"
```
Comment (ArchanaKakani) [inline oem/ibm/libpldmresponder/file_io_by_type.cpp:212]: same as above

File: oem/ibm/libpldmresponder/file_io.cpp [hunk 3]
```diff
@@ -298,8 +298,10 @@ Response Handler::readFileIntoMemory(const pldm_msg* request,
     auto fileSize = fs::file_size(value.fsPath);
     if (offset >= fileSize)
     {
-        std::cerr << "Offset exceeds file size, OFFSET=" << offset
-                  << " FILE_SIZE=" << fileSize << "\n";
+        std::cerr
+            << "fileIO:Handler::readFileIntoMemory:Offset exceeds file size, OFFSET="
+            << offset << " FILE_SIZE=" << fileSize << "FILE_HANDLE"
```
Comment (ArchanaKakani) [inline oem/ibm/libpldmresponder/file_io.cpp:303]: Add space before printing FILE_HANDLE

File: oem/ibm/libpldmresponder/file_io_by_type.cpp (hunk 1 above)
Comment (sagisin) [reply oem/ibm/libpldmresponder/file_io_by_type.cpp]: done

File: oem/ibm/libpldmresponder/file_io_by_type.cpp (hunk 2 above)
Comment (sagisin) [reply oem/ibm/libpldmresponder/file_io_by_type.cpp]: done

File: oem/ibm/libpldmresponder/file_io.cpp (hunk 3 above)
Comment (sagisin) [reply oem/ibm/libpldmresponder/file_io.cpp]: done

**Review by ArchanaKakani — APPROVED** [review]

**Review by jaypadath — APPROVED** [review]

**Pull request merged**